### ISSUE TITLE
Critical vulnerability in html-docx-js

### ISSUE THREAD

**Issue by theLeftTenant** [wnabil/ngx-export-as#34]
Npm Audit reveals a couple critical vulnerabilities ([1](https://www.npmjs.com/advisories/1066), [2](https://www.npmjs.com/advisories/1067)) related to "prototype pollution" in the version of lodash that html-docx-js uses. Html-docx-js has an [open issue](https://github.com/evidenceprime/html-docx-js/issues/107) requesting someone simply update lodash to the latest version, but there hasn't been any action in the html-docx-js repo since 2016.

A maintainer of another project that depends on html-docx-js was trying to find a [replacement for that dependency for this reason](https://github.com/cds-snc/simplify-privacy-statements/issues/143), but seems to have had trouble finding an alternative.

**Comment by wnabil** [comment]
Yes seems i will switch to use another package but i am really busy currently so any PR is welcome, there is already an opened issue that i think also related to this one as Html-docx-js becomes outdated
https://github.com/wnabil/ngx-export-as/issues/21

**Comment by theLeftTenant** [comment]
Yup. I actually cloned their project and performed the losash updates. In
order to test tho, I’d have to perform updates across the board... lots of
deprecated dev/dependencies and methods. I may send em what I got. But for
any hope of them accepting the PR I’m thinking I’d have to get creative.
Maybe reach them at their company email. I don’t think anyone there checks
that repos activity.

Adam
